### DISCUSSION TITLE
types of CSS

### DISCUSSION THREAD

**Discussion by DuajDiaz** [DuajDiaz/Travel-Website#49] [Q&A] [answered]
What are some of the types of CSS that are used?

**Comment by Bunny77K9** [comment, marked as answer]
There are three main types of CSS present:

    Inline CSS: Supports the addition of CSS inline, alongside HTML elements
    External CSS: Used to import an external CSS file to the HTML document
    Embedded CSS: Used to add CSS styles by making use of the <style> attribute
